src/main/java/com/google/api/codegen/util/NameFormatter.java

```diff
@@ -41,6 +41,11 @@
   String methodName(Name name);
 
   /**
+   * Formats the name as a field name.
+   */
+  String fieldName(Name name);
```

Comment:
This is another problem with differences in Go, since fields (member of structs) are exported, they end up in upper camel. Local variables, like function parameters, are in lower camel.

I think the use here is that class member variables should use field name, since they are logically a field (member) of a "struct".